app/controllers/projects_controller.rb

```diff
@@ -111,4 +109,8 @@ def projects_for_user
   def get_environments
     @environments = Environment.all
   end
+
+  def current_project
+    @project ||= (Project.find_by_param!(params[:id]) if params.try(:[], :id))
+  end
```

Comment:
Add a comment saying you're overriding the method in the `CurrentProject` module, due to the different param.